ow_plexil/src/plans/TestFaultDependencies.plp

```diff
@@ -0,0 +1,43 @@
+// The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+// Research and Simulation can be found in README.md in the root directory of
+// this repository.
+
+#include "plan-interface.h"
+
+// Tests the FaultDependencies class. Depending on the XML file used, 
+// the fault info of a subsystem will be printed out.
+
+
+TestFaultDependencies:
+{
+
+  PrintNode:
+  {
+    Repeat true;
+    Wait 3;
+
+    if !Lookup(IsOperable("Power")){
+      log_info("Power Subsystem Inoperable.");
+      log_info("Power Subsystem Faulty: ", Lookup(IsFaulty("Power")));
+      log_info("Active Faults In Subsystem: ", Lookup(ActiveFaults("Power")), "\n");
+    }
```

Comment:
Same problem for these statements when I activate a high power draw. No statement, and no warning message.

Reply:
The high power draw needs to be left on long enough to cause a Low State Of Charge fault. In the XML the Low State Of Charge and the Thermal Runaway are the only two faults able to cause the Power system to be inoperable.